mimium-lang/src/interner.rs

```diff
@@ -136,13 +136,13 @@ pub struct TypeNodeId(pub TypeKey);
 
 impl PartialEq for ExprNodeId {
     fn eq(&self, other: &Self) -> bool {
-        self.to_expr() == other.to_expr() && self.to_span() == self.to_span()
+        self.to_expr() == other.to_expr() && self.to_span() == other.to_span()
```

Comment:
Ok, it was comparing identical span....

Reply:
Aw, thanks for catching! This was my oversight in #27...